### PR TITLE
Updates http/report UI msg based on port value

### PR DESCRIPTION
**Pull request by danehans**
Fixes https://github.com/istio/fortio/issues/110

### CODE REVIEW

File: ui/uihandler.go [hunk 1]
```diff
@@ -845,7 +845,13 @@ func Serve(baseurl, port, debugpath, uipath, staticRsrcDir string, datadir strin
 	}
 	debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
 	http.HandleFunc(uiPath, Handler)
-	fmt.Printf("UI starting - visit:\nhttp://%s%s\n", urlHostPort, uiPath)
+	switch {
```
Comment (ldemailly) [inline ui/uihandler.go]: why a switch when this is an `if`/`else`

File: ui/uihandler.go [hunk 2]
```diff
@@ -845,7 +845,13 @@ func Serve(baseurl, port, debugpath, uipath, staticRsrcDir string, datadir strin
...
+	case strings.Contains(port, ":"):
+		fmt.Printf("UI starting - visit:\nhttp://%s%s\n", urlHostPort, uiPath)
+	default:
+		fmt.Printf("UI starting - visit:\nhttp://%s%s"+
```
Comment (ldemailly) [inline ui/uihandler.go]: let's keep the common part together, remove the \n and then just print "   (or any host/ip reaching this machine)\n" or "\n"

**Review by ldemailly — COMMENTED** [review]
assuming it's working / you tested :-)

**Review by ldemailly — APPROVED** [review]

**Comment by danehans** [comment]
I test the serv and repo UI's:

__Test 1:__ Serv UI:
```
$ docker run --name fortio --rm -p 8088:8088 -p 8079:8079 istio/fortio:webtest server -loglevel verbose -http-port 8088
<SNIP>
Https redirector running on :8081
UI starting - visit:
http://localhost:8088/fortio/   (or any host/ip reachable on this server)
19:04:07 I uihandler.go:91> Using resources directory set at link time: /usr/local/lib/fortio
Fortio 0.6.8 echo server listening on port :8088

$ docker run --name fortio --rm -p 8088:8088 -p 8079:8079 istio/fortio:webtest server -loglevel verbose -http-port 172.17.0.2:8088
Fortio 0.6.8 grpc ping server listening on port 8079
Https redirector running on :8081
UI starting - visit:
http://172.17.0.2:8088/fortio/
<SNIP>
Fortio 0.6.8 echo server listening on port 172.17.0.2:8088
```
__Test 2:__ Report UI:
```
$ docker run --name fortio --rm -p 8088:8088 -p 8079:8079 istio/fortio:webtest report -loglevel verbose -http-port 172.17.0.2:8088
<SNIP>
Browse only UI starting - visit:
http://172.17.0.2:8088/
Https redirector running on :8081

$ docker run --name fortio --rm -p 8088:8088 -p 8079:8079 istio/fortio:webtest report -loglevelerbose -http-port 8088
Browse only UI starting - visit:
http://localhost:8088/   (or any host/ip reachable on this server)
<SNIP>
```

**Comment by ldemailly** [comment]
Nice ! thanks!

**Pull request merged**